esprima.js

```diff
@@ -1274,27 +1281,13 @@
 
                     default:
                         if (isOctalDigit(ch)) {
```

Comment:
@michaelficarra I'm legitimately confused. Does the spec specify this as disallowed inside template strings or is it ambiguous. If you're telling me it's clear, I agree with you. Just not sure what to make of all the info.